### ISSUE TITLE
Possibility to enable/disable Bot-Defense with Endpoint Policy

### ISSUE THREAD

**Issue by Tobse95** [F5Networks/f5-appsvcs-extension#696]
### Is your feature request related to a problem? Please describe.
There is actually no way to enable/disable a bot defense profile in an endpoint policy.

### Describe the solution you'd like
To continue with our as3 migration, we need the possibility to disable Bot-Defense when a specific http uri is called.
The functionality is implemented for a waf policy:

https://clouddocs.f5.com/products/extensions/f5-appsvcs-extension/latest/declarations/application-security.html#endpoint-policy-with-default-rule-to-disable-waf
```

"endpointPolicy": {
              "class": "Endpoint_Policy",
              "rules": [
                  {
                    "name": "enableWAF",
                        "conditions": [{
                            "type": "httpUri",
                            "path": {
                                "values": ["example.com"]
                            }
                        }
                    ],
                    "actions": [{
                            "type": "waf",
                            "policy": {
                                "use": "wafPolicy"
                            }
                        }
                    ]
                  },
                  {
                    "name": "disableWAF",
                    "conditions": [],
                    "actions": [{
                            "type": "waf"
                        }
                    ]
                  }
```

We have the use case to disable the bot defense:
F.e.:
```
{
                    "name": "disableBotdefense",
                    "conditions": [],
                    "actions": [{
                            "type": "botdefense"
                        }
                    ]
                  }
```

The second use case is to enable a specific bot defense profile:
F.e.:

```
{
    "endpointPolicy": {
        "class": "Endpoint_Policy",
        "rules": [
            {
                "name": "enableBotdefense",
                "conditions": [
                    {
                        "type": "httpUri",
                        "path": {
                            "values": [
                                "example.com"
                            ]
                        }
                    }
                ],
                "actions": [
                    {
                        "type": "botdefense",
                        "policy": {
                            "use": "botdefense_profile"
                        }
                    }
                ]
            }
        ]
    }
}
```
          
      
### Describe alternatives you've considered
There are no useful alternatives.

**Comment by sunitharonan** [comment]
Thank you for reaching out. We have added this issue to our internal product backlog as AUTOTOOL-3768 to prioritize.

**Comment by sunitharonan** [comment]
Closing this as it is fixed in AS3 3.47
